third_party/move/move-vm/runtime/src/config.rs

```diff
@@ -12,6 +12,7 @@ pub const DEFAULT_MAX_VALUE_NEST_DEPTH: u64 = 128;
 pub struct VMConfig {
     pub verifier: VerifierConfig,
     pub max_binary_format_version: u32,
+    pub max_identifier_size: u32,
```

Comment:
How about we just include the whole `DeserializeConfig` as a field here? This way we can easily add new config options for the deserializer without having to touch the VM.